src/main/java/com/teamabnormals/abnormals_core/core/AbnormalsCore.java

```diff
@@ -5,6 +5,8 @@
 import com.teamabnormals.abnormals_core.core.registry.ACTileEntities;
 import com.teamabnormals.abnormals_core.core.util.registry.RegistryHelper;
 
+import net.minecraft.resources.IResourceManager;
```

Comment:
These two imports aren't consistent with the other imports in this class, they should be in the same areas as the other imports related to them.